### ISSUE TITLE
Add content title

### ISSUE THREAD

**Issue by Maelplaine** [libero/libero#2]
## Problem / Motivation
 
- WHO
As a reader
- WHEN and WHERE
when I look at the top of an article
- WHAT
I would like to be able to view the title of the article
- WHY
so that I know what I am reading

## Proposed solution and tasks

List of tasks for this ticket to be complete:
- [ ] A descriptive of a task
- [ ] A description

## Clarification needed and assumptions

- A question
- An assumption

## Technical notes
- We could use this technology 
- More notes or suggestions

**Comment by thewilkybarkid** [comment]
What should happen to titles that contain MathML when MathML isn't supported? (eg HTML's `<title>`)

**Comment by thewilkybarkid** [comment]
What does 'Inline figures are required for RSC for non-unicode characters' mean?

**Comment by diversemix** [comment]
RSC were explaining that character symbols in chemistry (like triple bond) are actually represented as inline images.

**Comment by thewilkybarkid** [comment]
Interesting, would be interested in seeing an example. Sounds like something to ignore for core now though, if they want to do that can add it in as a custom extension and maybe make it a Libero extension in the future.

**Comment by Maelplaine** [comment]
@thewilkybarkid yes we can definitely drop that for the MVP. I will update the ticket description.
